test/programs/test_multi_server.py

```diff
@@ -8,33 +8,26 @@ def test_resolve(self) -> None:
         p.ctx.player_names = {
             (1, 1): "AAA",
             (1, 2): "aBc",
-            (1, 3): "abC",
+            (1, 3): "a bc",
         }
```

Comment:
Can player name have underscore?

If so what happens if there's a player named `"a bc"` and another player named `"a_bc"`?

Reply:
https://github.com/ArchipelagoMW/Archipelago/pull/3283#issuecomment-2103772821